plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -445,10 +445,8 @@ class StaticHtmlReporter : Reporter {
             }
 
             tbody {
-                val projectRow = table.rows.single { it.id == project.id }
-                projectRow(project.id.toCoordinates(), 1, projectRow)
-                (table.rows - projectRow).forEachIndexed { rowIndex, pkg ->
-                    projectRow(project.id.toCoordinates(), rowIndex + 2, pkg)
+                table.rows.forEachIndexed { rowIndex, pkg ->
```

Comment:
Commit message: Could you go into a bit more detail what the "desired ordering" is? Am I reading the code right that the desire is to make the root project go first, and it's that logic that you're simplifying?

Reply:
Much better, thank you!